manuscript/manuscript.md

```diff
@@ -311,13 +310,17 @@ their grant guidelines to prevent sharing personally identifiable information,
 and to anonymize data on human subjects.
 
 In small datasets, a hashing scheme (anonymizing PII by converting it into
-a numeric key of a fixed length with a standard alogrithm) is enough to
+a numeric key of a fixed length with a standard algorithm) is enough to
 anonymize minimal personal information. Make sure to not store the hashing scheme
-with the data to prevent inadvertent sharing and don't use a commonplace hashing
+with the data to prevent inadvertent sharing and to not use a commonplace hashing
 technique. Famously, New York City officials shared what they thought was
 anonymized data on cab drivers and over 173 million cab rides. However, it was
 quickly recognized that the city anonymized the data with a simple MD5 hashing
 scheme and all 20 GB of data were de-anonymized in a matter of hours [@Goodin].
+This type of error can be prevented by asking a trusted colleague to try to
```

Comment:
Perhaps this would be a better (more academic, with broader context) citation that includes the taxi and many other examples?

Cormode, Graham. "The confounding problem of private data release." 18th International Conference on Database Theory. 2015.

[lipics-vol31-icdt2015-complete.pdf](https://github.com/emhart/10-simple-rules-data-storage/files/15785/lipics-vol31-icdt2015-complete.pdf)